elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/MetricFunction.java

```diff
@@ -30,6 +30,8 @@
 
     private String description;
 
+    private String category;
```

Comment:
WHy do MetricFunctions need categories - when the associated metric already has one?

Reply:
https://github.com/yahoo/elide/issues/1005#issuecomment-541196843 i saw this in the RFC